cpp/CMakeLists.txt

```diff
@@ -156,6 +156,24 @@ include(cmake/Modules/JitifyPreprocessKernels.cmake)
 # find cuFile
 include(cmake/Modules/FindcuFile.cmake)
 
+# Workaround until https://github.com/rapidsai/rapids-cmake/issues/176 is resolved
+if(NOT BUILD_SHARED_LIBS)
+  include("${rapids-cmake-dir}/export/find_package_file.cmake")
+  file(COPY "${CUDF_SOURCE_DIR}/cmake/Modules/FindcuFile.cmake"
```

Comment:
You shouldn't need this copy as you should be able to pass `rapids_export_find_package_file` the location of the file in your source directory